tests/php/test-functions.global.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Class WP_Test_Jetpack_Global_Functions file.
+ *
+ * @package Jetpack
+ */
+
+/**
+ * Class WP_Test_Jetpack_Global_Functions
+ */
+class WP_Test_Jetpack_Global_Functions extends WP_UnitTestCase {
+	/**
+	 * Tests for jetpack_get_site_suffix().
+	 *
+	 * @covers jetpack_get_site_suffix
+	 */
+	public function test_jetpack_get_site_suffix() {
+		$suffix = jetpack_get_site_suffix();
+		$this->assertSame( 'example.org', $suffix );
+
+		// TLD.
+		$suffix = jetpack_get_site_suffix( 'https://dortmund.com' );
+		$this->assertSame( 'dortmund.com', $suffix );
+
+		// Subdomain.
+		$suffix = jetpack_get_site_suffix( 'https://borussia.dortmund.com' );
```

Comment:
Could we use example domains, or at least ones owned by a8c?

Reply:
Fixed in e0c2e11

Reply:
e0c2e11d96ee64a891d9acb78440ab5709ff583f